ArtOfIllusion/src/artofillusion/material/LinearMaterialMapping.java

```diff
@@ -45,7 +45,8 @@ public String getName()
     return "Linear";
   }
 
-  public static boolean legalMapping(Object3D obj, Material mat)
+  @Override
+  public boolean legalMapping(Object3D obj, Material mat)
```

Comment:
I agree, we shouldn't make incompatible changes unless we have a really good reason to.  What's the reason in this case?